app/javascript/pages/Dashboard/Leaderboard/index.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useContext } from 'react'
+
+import * as R from 'ramda'
+import moment from 'moment'
+import styled from 'styled-components'
+import { useQuery } from '@apollo/react-hooks'
+import { Alert, Title } from '@zendeskgarden/react-notifications'
+import { MD, LG } from '@zendeskgarden/react-typography'
+import { Skeleton } from '@zendeskgarden/react-loaders'
+import { Tag } from '@zendeskgarden/react-tags'
+
+import LeaderboardQuery from './leaderboardQuery.gql'
+import ListItem from 'components/ListItem'
+import NamedAvatar from 'components/NamedAvatar'
+import OfficeFilter from 'components/OfficeFilter'
+import { UserContext } from 'context'
+
+const { zdSpacingXxs, zdColorGrey300 } = require('@zendeskgarden/css-variables')
+
+const SectionHeader = styled.div`
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid ${zdColorGrey300};
+  margin-bottom: ${zdSpacingXxs};
+  padding-bottom: ${zdSpacingXxs};
+`
+const SectionTitle = styled(LG)`
+  align-self: flex-end;
+`
+
+const Loader = _props => (
+  <>
+    {Array(10)
+      .fill(null)
```

Comment:
is the `null` necessary? - I think it still works if you map over undefined?

Reply:
yeah it should still work.